scheduler/bin/run-docker.sh

```diff
@@ -33,6 +33,7 @@ docker run \
     --publish=${COOK_PORT}:${COOK_PORT} \
     -e "COOK_PORT=${COOK_PORT}" \
     -e "COOK_NREPL_PORT=${COOK_NREPL_PORT}" \
+    -e "COOK_FRAMEWORK_ID=cook-framework-${COOK_PORT}" \
```

Comment:
Reading that now, I realize there are two interpretations.

What I meant is that if we make the framework id configurable and don't read the framework id from zookeeper, then users that upgrade will get bit if they don't know to configure the framework id.